### PR TITLE
Clean up the README and docs

### LINKED ISSUE #68: Update the README
I don't think we need to duplicate the manual in the README, which is what it's currently kind of doing.

Instead, it might be good to use the README to spell out the intended scope of this package.

### PR DESCRIPTION
**Pull request by mortenpi**
Instead of deleting the examples that were in the readme, I moved them onto the manual landing page. I think that's actually a good place for stuff like that.

I did create a small usage example into the README. It looks like this now: https://github.com/JuliaAtoms/AtomicLevels.jl/blob/mp/readme/README.md

Close #68

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+As a simple usage example, constructing a configuration for an S-like state with an open `3p` shell looks like
+
+```julia-repl
+julia> configuration = c"[Ne]* 3s2 3p4"
+1s² 2s² 2p⁶ 3s² 3p⁴
+```
+
+which is of type `Configuration`. To access information about subshells, you can index into
+the configuration which returns a tuple. Tuple contains an `Orbital` object, so you
```
Comment (jagot) [inline README.md:36]: "Tuple contains an..." is missing an article, i.e. "The tuple contains an..."

Comment (mortenpi) [reply README.md:36]:
```suggestion
the configuration which returns a tuple. The tuple contains an `Orbital` object, so you
```

File: src/levels.jl [hunk 2]
```diff
@@ -1,3 +1,13 @@
+# * Level
+
+"""
+    Level(csf, J)
+
+Given a [`CSF`](@ref) with a final [`Term`](@ref), a `Level`
+additionally specifies a total angular momentum ``J``. By further
+specifying a projection quantum number ``M_J``, we get a
+[`State`](@ref).
```
Comment (mortenpi) [inline src/levels.jl:9]: Hmm. So the CSF here would have to be in LS coupling then?

Comment (jagot) [reply src/levels.jl:9]: No, not true; in the `jj`-coupled case, there is however trivially only one `Level`, with the same value of `J` as the CSF.

Comment (jagot) [reply src/levels.jl:9]: See https://github.com/JuliaAtoms/AtomicLevels.jl/pull/91/files/3596028a910bd6648aa32e3247b5a31bb1801125#diff-800c4b457b4d08bf3b3b3abaf4d29a4fe9e83ede3e72a22397948085e9ab7c74R80

Comment (mortenpi) [reply src/levels.jl:9]: Ah, true that. While trivial, nothing stopping you from constructing a `Level` object in the `jj` case.

File: src/levels.jl [hunk 3]
```diff
@@ -22,11 +50,73 @@ Base.:(==)(l1::Level, l2::Level) = ((l1.csf == l2.csf) && (l1.J == l2.J))
...
+julia> csfs(rc"1s 2p")
+2-element Vector{RelativisticCSF{RelativisticOrbital{Int64}, Seniority}}:
+ 1s(₁1/2|1/2) 2p(₁3/2|1)-
+ 1s(₁1/2|1/2) 2p(₁3/2|2)-
+
+julia> csfs(rc"1s 2p-")
+2-element Vector{RelativisticCSF{RelativisticOrbital{Int64}, Seniority}}:
+ 1s(₁1/2|1/2) 2p-(₁1/2|0)-
+ 1s(₁1/2|1/2) 2p-(₁1/2|1)-
+```
```
Comment (mortenpi) [inline src/levels.jl]: These doctests don't actually call `levels`?

Comment (jagot) [reply src/levels.jl]: Should be fixed now

**Pull request merged**